aws/windows-agent/main.tf

```diff
@@ -35,19 +31,20 @@ module "dcos" {
   dcos_variant = "open"
 
   dcos_version              = "1.13.0"
-  ansible_bundled_container = "sebbrandt87/dcos-ansible-bundle:windows-support"
+  //TO DO : change a tag once a new build of docker image available
+  ansible_bundled_container = "mesosphere/dcos-ansible-bundle:feature-windows-support-039d79d"
 
   # provide a SHA512 hashed password, here "deleteme"
   dcos_superuser_password_hash = "$6$rounds=656000$YSvuFmasQDXheddh$TpYlCxNHF6PbsGkjlK99Pwxg7D0mgWJ.y0hE2JKoa61wHx.1wtxTAHVRHfsJU9zzHWDoE08wpdtToHimNR9FJ/"
   dcos_superuser_username      = "demo-super"
 
-  additional_windows_private_agent_ips       = ["${concat(module.winagent.private_ips)}"]
-  additional_windows_private_agent_passwords = ["${concat(module.winagent.windows_passwords)}"]
+  additional_windows_private_agent_ips       = ["${concat(module.windows-instance.private_ips)}"]
+  additional_windows_private_agent_passwords = ["${concat(module.windows-instance.windows_passwords)}"]
 }
 
-module "winagent" {
+module "windows-instance" {
   source  = "dcos-terraform/windows-instance/aws"
-  version = "~> 0.2.1"
+  version = ">= 0.0.1"
```

Comment:
a pessimistic version constraint like `~> 0.0.1` would match any "alpha" version and would exclude higher versions like `0.1.0` or `0.2.0` etc.